components/phpunit_bridge.rst

```diff
@@ -133,6 +133,28 @@ completely disable the deprecation helper. This is useful to make use of the
 rest of features provided by this component without getting errors or messages
 related to deprecations.
 
+Write Assertions about Deprecations
+-----------------------------------
+
+When adding deprecations to your code, you might like writting tests that verify
+that they are triggered as required. To do so, the bridge provides the
+``@expectedDeprecation`` annotation that you can use on your test methods,
+alongside with the expected message, given in the same format than for the
+`PHPUnit assertStringMatchesFormat()`_ method). If you expect more than one
+deprecation messages for a give test method, you can use the annotation several
+times::
+
+    /**
+     * @group legacy
+     * @expectedDeprecation This "%s" method is deprecated.
+     * @expectedDeprecation The second argument of the "%s" method is deprecated.
```

Comment:
is the order of annotations important ? If they must be in the same order than the triggered deprecation, this should be documented (or changed in the code)